### PR TITLE
Pd 479 create ddns migration article

### PR DESCRIPTION
**Pull request by micjohnson777**
This PR creates a new t
utorial article with instructions on migrating from the SCALE DDNS service to the new ddns-updater application.

This merges into the 22.12.3 branch. 
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Comment by micjohnson777** [comment]
This PR is ready for an initial review for all feedback before a final PR for merging is submitted.

File: content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+{{< trueimage src="/images/SCALE/22.12/ddnsUpdaterAppWidget.png" alt="ddns-updater Application Widget" id="1 ddns-updater Application Widget" >}}
+
+Click **Install** to open the **ddns-updater** configuration wizard.
+
+Select the timezone that applies to your server location from the **Timezone** dropdown list.
+
+Click **Add** to the right of **DNS Provider Configuration** to display provider setting options. 
+Select the DDNS provider from the **Provider** dropdown list. 
+Each provider displays the settings required to establish a connection with and authenticate to that provider.
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md:44]: Would this be clearer with something like:

Once a provider is selected, the **DNS Provider Configuration** box will display the settings fields required to establish a connection with and authenticate to that provider.

File: content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+{{< hint type=info >}}
+The SCALE service dynamic DNS list includes providers not on the ddns-Updater application list. 
+If the service configuration uses a service not in the application, create a new account with a provider on the list in the ddns-updater application.
+{{< /hint >}}
+
+The update period default for the SCALE service is different from the application. Accept the default settings.
+
+{{< trueimage src="/images/SCALE/22.12/ddns-updaterConfigPeriodSettings.png" alt="ddns-updater Provider Settings" id="3 Add ddns-updater Provider Settings" >}}
+
+Add the address(es) for the [shoutrrr gopher notification service(s)](https://containrrr.dev/shoutrrr/0.7/services/overview/) you want to use. Separate each entry with a comma.
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md:57]: Is configuring shoutrrr notifications required or optional? If optional, maybe an _If... any_ would be better than _Add... the_

File: content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+Enter either the IPv4 or IPv6 HTTP public provider and the public fetcher IP addreses you want to use or select **All** in each field.
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md:59]:
```suggestion
Enter the IPv4 or IPv6 HTTP public provider(s) and the public IP fetcher you want to use or select **All providers** in each field.
```

File: content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+Select **Host Network** to bind to the host network, but we recommend leaving this disabled.
+
+Select the **DDNS Updater Data Storage** option from the **Type** dropdown list. Options are the iXVolume or host path.
+
+Click **Save**.
+
+{{< taglist tag="scaleddns" limit+"10" title="Related Dynamic DDNS" >}}
+
+
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md:71]:
```suggestion
```

File: content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
+---
+title: "Migrating from Dynamic DNS Service to DDNS Updater App"
+description: "This article provides instructions for users migrating from the SCALE Dynamic DNS service to the new ddns-updater application." 
+weight: 15
```
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md:4]: actually, I need to change this weight to get it in the right order.

File: content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+aliases:
+tags:
+- scaleddns
+---
+
+
+{{< include file="content/_includes/SCALEServiceDeprecationNotice.md" type="page" >}}
```
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md:11]: I'll ponder this.

File: content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md (hunk 1 above)
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md:44]: never use the word will in a procedure document. It either does or does not do something. Keep to present tense active voice. :-)

File: content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md (hunk 2 above)
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md:57]: It is unclear if it is optional so I included the instructions.

File: content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/SystemSettings/Services/DDNSServiceMigrate.md:11]: The deprecation applies to the service not the application so in this instance, the deprecation notice applies only to the service but I might remove this notice about the deprecated service or use a different snippet to inform users of the change

File: content/Solutions/Integrations/SMBClustering.md [hunk 7]
```diff
@@ -133,14 +151,19 @@ Attempting to manage the share from the SCALE UI is not recommended.
...
-4. Browse to the cluster volume folder to view or modify files.
+1. Connected to the same network as the clustering environment, open **File Explorer**.
+   
+   ![WindowsFileExplorereAccessClusterShare](/images/TrueCommand/2.2/WindowsFileExplorereAccessClusterShare.png "Cluster Volume Share Options")
 
-![WindowsFileExplorereAccessClusterShare](/images/TrueCommand/2.2/WindowsFileExplorereAccessClusterShare.png "Cluster Volume Share Options")
+2. Clear the contents and enter `\\` followed by the IP address or host name of one of the clustered SCALE systems in the **Navigation** bar. 
+   Press <kbd>Enter</kbd>.
+
+3. Enter the user name and password for an Active Directory user account when prompted.
```
Comment (DjP-iX) [inline content/Solutions/Integrations/SMBClustering.md:163]:
```suggestion
3. Enter the username and password for an Active Directory user account when prompted. 
```

**Review by DjP-iX — CHANGES_REQUESTED** [review]

**Review by DjP-iX — APPROVED** [review]

**Pull request merged**